test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -36,11 +36,35 @@ public void elementsAreUnique() throws Exception {
         assertNotUnique(null, "a", "b", null);
     }
 
+    @Test
+    public void elementsHaveNull() throws Exception {
+        // empty list
+        assertNoneNull();
+
+        // has null
+        assertAnyNull((Object) null);
+        assertAnyNull(null, "CS2103");
+        assertAnyNull(null, null);
+
+        // none null
+        assertNoneNull("5");
+        assertNoneNull("CS2103", 5);
+        assertNoneNull(2103);
+    }
+
     private void assertAreUnique(Object... objects) {
         assertTrue(Utils.elementsAreUnique(Arrays.asList(objects)));
     }
 
     private void assertNotUnique(Object... objects) {
         assertFalse(Utils.elementsAreUnique(Arrays.asList(objects)));
     }
+
+    private void assertAnyNull(Object... objects) {
+        assertTrue(Utils.isAnyNull(objects));
+    }
+
+    private void assertNoneNull(Object... objects) {
+        assertFalse(Utils.isAnyNull(objects));
+    }
```

Comment:
It's good to follow the style used by the code nearby 👍
Having said that, it is doubtful if extracting these two methods add any value.
You could use the assertFalse and assertTrue inline as there are no conversions needed unlike in the elementsAreUnique method.